src/main/resources/the_warlordResources/localization/eng/WarlordMod-Card-Strings.json

```diff
@@ -149,8 +149,7 @@
   },
   "the_warlord:Ablution": {
     "NAME": "Ablution",
-    "DESCRIPTION": "Remove all of your debuffs.",
-    "UPGRADE_DESCRIPTION": "Remove all of your debuffs. NL Gain !M! Artifact."
+    "DESCRIPTION": "Remove all of your debuffs. NL Gain !B! Block for each debuff removed."
```

Comment:
As stated in the Ablution class, I'd like to see 
"EXTENDED_DESCRIPTION": [" NL (You have !M! debuffs.)"]